lib/src/lib.rs

```diff
@@ -157,15 +157,14 @@ mod inner_test {
     use crate::bitcoin::Network;
     use crate::{iterate, Config};
     use std::sync::mpsc::sync_channel;
+    use test_log::test;
```

Comment:
didn't know about this, it's great!
Ideally would have been better to introduce this in a separate commit